### PR TITLE
Solidity contract: add missing files on bbgo contract folder

### PR DESCRIPTION
**Pull request by zenixls2**
- add lint on solidity files and js files
- add test
- add npm scripts
- update README
- import used contracts from https://github.com/maticnetwork/pos-portal/ directly and add solhint disables

### CODE REVIEW

**Comment by zenixls2** [comment]
also should we make it Pausable to prevent possible contract issues in the future?  
ex: https://github.com/mathwallet/BSC-Contracts/blob/main/Contracts/BEP20.sol

**Comment by zenixls2** [comment]
refer: https://docs.matic.today/docs/develop/ethereum-matic/mintable-assets

File: contracts/contracts/child/ChildToken/ChildMintableERC20.sol [hunk 1]
```diff
@@ -0,0 +1,79 @@
+pragma solidity 0.6.6;
```
Comment (c9s) [inline contracts/contracts/child/ChildToken/ChildMintableERC20.sol:1]: is this just moving file or you build a new file?

**Review by c9s — APPROVED** [review]

File: contracts/contracts/child/ChildToken/ChildMintableERC20.sol (hunk 1 above)
Comment (zenixls2) [reply contracts/contracts/child/ChildToken/ChildMintableERC20.sol:1]: Copy from matic repo

**Comment by zenixls2** [comment]
@c9s add comments in README

**Review by c9s — APPROVED** [review]

**Pull request merged**